neuroml/nml/generatedssupersuper.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env python3
+"""
+Super class for GeneratedsSuper
+
+File: neuroml/nml/generatedssupersuper.py
+
+Copyright 2022 NeuroML contributors
+"""
+
+
+class GeneratedsSuperSuper(object):
```

Comment:
@pgleeson : just an FYI: I've moved the common helper functions to this super class for generatedssuper, so all libNeuroML classes should now have them (since even if they don't extend `BaseWithoutId`, they will extend `GeneratedsSuper`, which extends `GeneratedsSuperSuper`).